### ISSUE TITLE
ASM Errors

### ISSUE THREAD

**Issue by YingboMa** [andreasnoack/JuliaBlas.jl#4]
Both `gemmFull.jl` and `gemmPrefetch.jl` will produce the following errors.

```julia
julia> using LC

julia> A = rand(5,5); C = similar(A);

julia> LC.gemm_nn(5,5,5,1.,A,1,5,A,1,5,0.,C,5,1);
<inline asm>:24:2: error: invalid symbol redefinition
        .DLOOP:
        ^
<inline asm>:133:2: error: invalid symbol redefinition
        .DCONSIDERLEFT:
        ^
<inline asm>:136:2: error: invalid symbol redefinition
        .DLOOPLEFT:
        ^
<inline asm>:167:2: error: invalid symbol redefinition
        .DPOSTACCUMULATE:
        ^
LLVM ERROR: Error parsing inline asm
```

**Comment by andreasnoack** [comment]
Hm. This used to work. Wondering what is going on. I will probably not have time to look at it today.

**Comment by YingboMa** [comment]
Closed by #3
